app/models/user.rb

```diff
@@ -9,4 +9,24 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  has_many :friendships
+  has_many :reverse_friendships, foreign_key: :friend_id, class_name: Friendship.name
```

Comment:
This can be written like this `has_many :reverse_friendships, foreign_key: :friend_id, class_name: "Friendship"`